### PR TITLE
login and signup button added

### PR DESCRIPTION
**Pull request by Abhinaba006**
## What is the change?
Remove this line and add a description

## Related issue?
close: #301 

## How was it tested?
Remove this line and tell us how the code was tested.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.
image

### CODE REVIEW

**Comment by Pranav016** [comment]
I think login button was already there and signup button was added here..

File: views/partials/header.ejs [hunk 1]
```diff
@@ -70,60 +55,46 @@
...
-          </ul>
+                        </li>
+                        <li id="sign-up">
+                            <a href="/sign-up" class="btn btn-info btn-sm text-white" role="button">
+              Sign up
+            </a>
+                        </li>
+                        <% } %>
+                            </ul>
+            </div>
```
Comment (Pranav016) [inline views/partials/header.ejs:86]: Code has a lot of indententation issues..

**Review by Pranav016 — COMMENTED** [review]
Will accept PR after resolving indentation issues but make sure next time you don't commit on your master..

File: views/partials/header.ejs (hunk 1 above)
Comment (Abhinaba006) [reply views/partials/header.ejs]: I have corrected the indentation and will taker care of making a new branch for working on a new issue.

**Comment by Abhinaba006** [comment]
@Pranav016 sir, please check, I have corrected the code.

**Comment by Pranav016** [comment]
/gssoc/

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**